lib/react-shim.js

```diff
@@ -0,0 +1,2 @@
+import * as React from 'react'
+export { React }
```

Comment:
this code is automatically injected by esbuild to avoid having to import `React` in every files.